src/ApparentHorizons/StrahlkorperGr.hpp

```diff
@@ -468,4 +468,18 @@ double irreducible_mass(double area);
  */
 double christodoulou_mass(double dimensionful_spin_magnitude,
                           double irreducible_mass);
+
+/*!
+ * \ingroup SurfacesGroup \brief Radial distance between two `Strahlkorper`s.
```

Comment:
Doxygen rendering is screwed up here, probably just because of the missing line break.

Reply:
Thanks for catching @wthrowe ... not sure how I missed that, but just pushed and squashed a fix